docs/en/latest/getting-started.md

```diff
@@ -202,25 +221,25 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f
 }'
 ```
 
-This configuration means that it will forward all matching inbound requests to the upstream service (`httpbin.org:80`) if they meet these specified criterion.
+This configuration means that it will forward all matching inbound requests to the Upstream service (`httpbin.org:80`) if they meet these specified criterion:
 
 - The HTTP method of the request is `GET`.
 - The request header contains the `host` field, and its value is `example.com`.
 - The request path matches `/anything/*`. `*` means any sub path. For example `/anything/foo?arg=10`.
 
-Now that the Route has been created, we can access the Upstream service from the address exposed by Apache APISIX.
+With the Route has created, we can access the Upstream service from the address exposed by APISIX:
 
 ```bash
 curl -i -X GET "http://127.0.0.1:9080/anything/foo?arg=10" -H "Host: example.com"
 ```
 
-This request will be forwarded to `http://httpbin.org:80/anything/foo?arg=10` by Apache APISIX.
+This request will be forwarded to `http://httpbin.org:80/anything/foo?arg=10` by APISIX.
```

Comment:
I did not get the expected response according to this configuration. The results are as follows:
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/30611190/165421954-add58487-0af5-42ed-b8a9-389f720c299c.png">
When I changed the Hostname in the Dashboard configuration to `Use the domain or IP from Node List`, it worked fine
<img width="758" alt="image" src="https://user-images.githubusercontent.com/30611190/165422361-776ca57a-10a5-431c-980f-28931b32e31b.png">

Reply:
Oops. I will fix the second Route creation step.

Reply:
Hi @navendu-pottekkat, I still can't get the correct response in this step. After adjusting the configuration information the correct return is obtained:
```
curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f136f87ad84b625c8f1" -X PUT -d '
{
  "methods": ["GET"],
  "host": "example.com",
  "uri": "/anything/*",
  "upstream": {
    "type": "round-robin",
    "pass_host": "node",
    "nodes": {
      "httpbin.org:80": 1
    }
  }
}'
```

Reply:
We pass `example.com` as header from the client. It is mentioned in this step. So, `pass_host` should be set to the default value of `pass`. Is that correct?

![Screenshot 2022-04-28 at 5 49 48 PM](https://user-images.githubusercontent.com/49474499/165750818-a888c51c-2db8-4882-a9c0-765b8c84afb0.png)

Reply:
Okay maybe I made a mistake somewhere.